python/shotgun_desktop/startup.py

```diff
@@ -30,7 +30,7 @@
 # Grab the name and the executable
 executable_name, ext = os.path.splitext(os.path.basename(sys.executable or ""))
 
-# If the executable is not named Shotgun or ShotGrid, then we need to patch sys.executable.
+# If the executable is not named Shotgun or Flow Production Tracking, then we need to patch sys.executable.
 if executable_name.lower() not in ["shotgun", "shotgrid"]:
```

Comment:
The change in the previous line is not accurate if you don't change this line. 

I think this change is out of topic here and should be covered by SG-29486.